pkg/nuget/packages_props/parse.go

```diff
@@ -0,0 +1,92 @@
+package config
+
+import (
+	"encoding/xml"
+	"golang.org/x/xerrors"
+
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"github.com/aquasecurity/go-dep-parser/pkg/utils"
+)
+
+	
+
+
+type propsPackageEntry struct{
```

Comment:
I think we can omit `props` and `propsPackage` for structs.
```suggestion
type entry struct{
```

Reply:
Can you do the same for other structures?